query/stdlib/experimental/to_test.go

```diff
@@ -85,53 +83,6 @@ from(bucket:"mydb")
 	}
 }
 
-func TestToOpSpec_BucketsAccessed(t *testing.T) {
```

Comment:
I guess the `BucketsAccessed` stuff is no longer needed?

Reply:
I couldn't find it being used anywhere.